src/jinja2/utils.py

```diff
@@ -846,8 +961,10 @@ def __new__(cls, base, encoding=None, errors="strict"):  # type: ignore
 
 def escape(s: t.Any) -> str:
     warnings.warn(
-        "'jinja2.escape' is deprecated and will be removed in Jinja"
-        " 3.1. Import 'markupsafe.escape' instead.",
+        "'jinja2.Markup' is deprecated and will be removed in Jinja"
```

Comment:
What to do with this warning? Simply remove it, as it is supposed to be removed in 3.1 anyway?

Reply:
Yes, this will be removed in 3.1, so you can omit any changes to it from your PR.